pkg/tasks/prometheusoperator_user_workload.go

```diff
@@ -41,6 +43,7 @@ func (t *PrometheusOperatorUserWorkloadTask) Run(ctx context.Context) error {
 		return t.create(ctx)
 	}
 
+	klog.V(3).Infof("UWM Prometheus is disabled (because UWM is disabled), existing related resources are to be destroyed.")
```

Comment:
```suggestion
	klog.V(3).Infof("UWM Prometheus Operator is disabled (because UWM is disabled), existing related resources are to be destroyed.")
```

Reply:
Good catch